<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PKGBUILDASSISTANT</class>
 <widget class="QMainWindow" name="PKGBUILDASSISTANT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PKGBUILD Assistant</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab_baseinfo">
       <attribute name="title">
        <string>基础信息</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_source">
       <attribute name="title">
        <string>源</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_relations">
       <attribute name="title">
        <string>相关性</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_depends">
       <attribute name="title">
        <string>依赖</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_function">
       <attribute name="title">
        <string>函数</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_create">
       <attribute name="title">
        <string>创建</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
